src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -186,14 +204,16 @@ def merge_evidence_qc_table(
     df_melt_insert_size = read_melt_insert_size(filename_melt_insert_size)
 
     # all data frames
-    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
-           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+    dfs = [df_ploidy, df_sex_assignments, df_bincov_median, df_wgd_scores, df_non_diploid,
+           df_manta_high_outlier, df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
            df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers,
            df_melt_insert_size]
     for df in dfs:
         df[ID_COL] = df[ID_COL].astype(object)
     output_df = reduce(lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs)
     output_df = output_df[output_df[ID_COL] != EMPTY_OUTLIERS]
+    output_df = output_df.replace([None, np.nan], 0.0)
+    output_df.rename(columns={ID_COL: NEW_ID_COL}, inplace=True)
```

Comment:
Is there a reason for making this change after the fact, rather than updating the value of ID_COL?

Side note that it may be worth asking around if people have strong feelings about using # to indicate the header line in this file. I don't think it's necessary for this file, and I think you have the right idea about updating it for concordance with other files and ease of use with pandas, but it is a common convention in the field, which is why the original header had #ID.

Reply:
Yes, this was done mainly so that upstream files did not have to be changed to the new ID naming standard.

Now that you mention that this is a common convention though, I'm wondering if we can/should just leave it as is?

Reply:
Updated all upstream tables to use `sample_id` - have hence removed this renaming from the script.